internal/files/repo_files.go

```diff
@@ -65,3 +73,41 @@ func (r *sqlRepository) Cancel(ctx context.Context, fileID string) error {
 	}
 	return nil
 }
+
+type spannerRepository struct {
+	client *spanner.Client
+}
+
+func (r *spannerRepository) Record(ctx context.Context, file AcceptedFile) error {
+	ctx, span := telemetry.StartSpan(ctx, "files-record", trace.WithAttributes(
+		attribute.String("achgateway.file_id", file.FileID),
```

Comment:
Can we add an attribute `achgateway.database` of `spanner` and `mysql` to these spans?

Reply:
Yep, I'll get that in here.